### PR TITLE
Add tooltips for Dropdowns

### LINKED ISSUE #254: Add context to delegate(?) icon and popover menu
image

It's not clear what this means or what it refers to, even when you click and open it up.

- [ ] add a tooltip, similar to the tooltip on the Seed node button
image

---

- [ ] add some kind of context or text here saying what any of this means. i'm on a project called "opera codex" then i click this fork icon to find out what it means and it says "swanux" and a bunch of characters and "delegate". hovering just shows the long string of characters as a tooltip, and clicking just opens the same "opera codex" project but with the big number appended to the end of the project name, and now "swanux delegate" is stuck to the fork. no idea what any of this means. But a sentence on the pop over on how I should use this or what I need it for should help!
image

### PR DESCRIPTION
**Pull request by sebastinez**
This PR closes #254

Adds a way to pass tooltip text to the `Dropdown` component, and more explanations to the Dropdown Trigger button

### CODE REVIEW

File: src/Dropdown.svelte [hunk 1]
```diff
@@ -32,9 +32,9 @@
 </style>
 
 <div class="dropdown">
-  {#each items as {key, value, badge}}
-    {#if key && value}
-      <div class="dropdown-item" class:selected={value === selected} on:click={() => onSelect(value)} title={value}>{@html key}
+  {#each items as {key, value, badge, title}}
+    {#if key && value && title}
```
Comment (cloudhead) [inline src/Dropdown.svelte:36]: ?

Comment (sebastinez) [reply src/Dropdown.svelte]: Hmm yeah we could make `title` required as an accessibility feature, so we don't have to check it here.

File: src/base/projects/PeerSelector.svelte [hunk 2]
```diff
@@ -11,15 +11,20 @@
 
   let meta: Peer | undefined;
   // List of items to be created for the Dropdown component.
-  let items: { key: string; value: string; badge: string | null }[] = [];
+  let items: { key: string; value: string; title: string; badge: string | null }[] = [];
+
+  function createTitle(p: Peer): string {
+    const name = p.person?.name ? p.person.name : p.id;
+    return p.delegate ? `${name} is the delegate peer of this project` : `${name} is a tracked peer by the delegate`;
```
Comment (cloudhead) [inline src/base/projects/PeerSelector.svelte:18]:
```suggestion
    return p.delegate ? `${name} is a delegate of this project` : `${name} is a peer tracked by this seed`;
```

Comment (sebastinez) [reply src/base/projects/PeerSelector.svelte]: Done in [827163c](https://github.com/radicle-dev/radicle-interface/pull/331/commits/827163c6a93d470825a3d8973c3dc5c3f0d38e82)

**Review by cloudhead — APPROVED** [review]

**Pull request merged**